tfjs-backend-webgpu/src/conv2d_mm_webgpu.ts

```diff
@@ -38,17 +38,16 @@ export class Conv2DMMProgram implements WebGPUProgram {
   hasPreluActivationWeights: boolean;
   fitA: boolean;
   fitB: boolean;
+  isChannelsLast: boolean;
 
   constructor(
       convInfo: backend_util.Conv2DInfo, addBias = false,
       activation: backend_util.Activation = null,
       hasPreluActivationWeights = false) {
     this.outputShape = convInfo.outShape;
-
-    util.assert(
-        convInfo.dataFormat === 'channelsLast',
-        () => 'TODO: NCHW is unimplemented');
-    this.dispatchLayout = {x: [3], y: [1, 2], z: [0]};
+    this.isChannelsLast = convInfo.dataFormat === 'channelsLast';
+    this.dispatchLayout = this.isChannelsLast ? {x: [3], y: [1, 2], z: [0]} :
+                                                {x: [1], y: [2, 3], z: [0]};
```

Comment:
Yes, it will be done in next PR. The reason I don't do it here is that I need to do it together with matmul one. Currently when transpose is true, matmul exists the similar issue. We can use the similar algorithm to solve both problems. And it's easier for review and perf collection.